src/main/java/net/commments/Test4.java

```diff
@@ -0,0 +1,37 @@
+import org.hamcrest.MatcherAssert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class Test4 {
+
+    private final WebDriver[] driver = new WebDriver[1];
+
+    @Test
+    public void test() {
+        this.driver().get("http://commentssprintone.azurewebsites.net");
+        this.driver().findElements(By.name("SelectedId")).get(0).click();
+        this.driver().findElement(By.xpath("//*[@value=\"Delete\"]")).click();
+        this.driver().findElement(By.xpath("//span[text()=\"Yes\"]")).click();
+        final String page = this.driver().getPageSource();
+        MatcherAssert.assertThat("Comment Text 0 is present", !page.contains("Comment Text 0"));
+    }
+
+    @BeforeMethod
+    public void createDriver() {
+        System.setProperty("webdriver.chrome.driver", "C:\\Automation\\src\\main\\resources\\chromedriver.exe");
```

Comment:
Please add driver to the PATH instead this configuration. See https://developers.thomsonreuters.com/sites/default/files/How%20To%20Add%20ChromeDriver%20To%20System%20Variables_0.pdf for details.

Reply:
@extsoft  done, pushed